src/shared/vpick.c

```diff
@@ -682,6 +682,22 @@ int path_pick_update_warn(
         return 1;
 }
 
+int path_pick_raw_or_dir(
```

Comment:
this really should carry the word "image" in the name, since this is only about DDIs after all, and references pick_filter_image_raw/pick_filter_image_dir after all